src/Analyser.php

```diff
@@ -478,7 +372,7 @@ private function detectFileByClassLoader(string $usedSymbol): ?string
         }
 
         try {
-            return $this->realPath($filePath);
+            return $this->trimPharPrefix($this->realPath($filePath));
```

Comment:
Is this even possible to happen? I think composer autoloader cannot resolve into phar.

Reply:
No idea. I just kept existing logic where `trimPharPrefix` was called in both cases, without thinking about it. But I think you're right and Composer ClassLoader can never return phar path, so I removed the call.